db/migrate/20151106130955_update_miq_shortcuts.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateMiqShortcuts < ActiveRecord::Migration
+  def up
+    MiqShortcut.find_each do |row|
```

Comment:
@Fryguy See my main comment above . . . agree, I don't think this migration is needed.